backend/utils/chat_notification.py

```diff
@@ -0,0 +1,218 @@
+from NovuPy.events import Events
+from NovuPy.subscribers import Subscribers
+from utils.room_utils import get_room_members, get_room
+from fastapi import HTTPException
+
+subscriber = Subscribers()
+event = Events()
+
+class Notification:
+    """
+    A class that handles the Novu instance message notification 
+    trigger for Channels, DM and when user is tagged.
+    This class is called after the message instance or object has been created
+    """  
+
+    async def check_message_tag(
+        self, message_obj, payload, 
+        sender_id
+        ):
+        """
+        A method that creates a notification trigger for tagged users
+        Args:
+            message_object-> Dict
+            payload-> Dict: message data that should be passed to novu
+            sender_id -> str
+        Raise:
+            HTTP_422- when Novu couldn't send notification to tagged users
+        """
+        tagged_users = []
+        # user_msg_tag = message_obj["richUiData"]['entityMap']
+        user_msg_tag = message_obj.get("entityMap", " ")
+        if user_msg_tag:
+            for i in range(len(user_msg_tag)):
```

Comment:
you can't use `'i'` to name a variable in a codebase like this, it is confusing and can be misleading. I suggest you use a proper variable name that clearly represents the data held by this variable.

Reply:
everything fixed now boss @mikengr